### DISCUSSION TITLE
Сохранение вещей копам

### DISCUSSION THREAD

**Discussion by Ggman77** [octothorp-team/public-dobrograd#678] [Предложения]
Здраствуйте, хочу предложить идею.
   Как вы знаете копам нельзя ставить храны в пд, а терять вещи не хочется, из за этого хочу предложить что бы когда вы брали форму и снаряжение, вещи скидывались в ящик(где лежит снаряга офицера).

**Comment by Dawaren** [comment]
Так пусть коп поставит свой хран дома.

**Comment by LENFYS** [comment]
Против

**Comment by travka4000** [comment]
Незнаю я за выглядит умно, тем более когда ты коп ты не можешь посещать свой дом часто где и будет хран лежать, а краймеры то еще те охотники за лутом и они точно взломают тебя. Я за
